lib/prefix.c

```diff
@@ -582,6 +607,23 @@ int prefix_match_network_statement(const struct prefix *n,
 }
 
 /* Copy prefix from src to dest. */
+void prefix_opaque_copy(struct prefix *dest, const struct prefix *src)
```

Comment:
This belongs in prefix_copy() and prefix_copy needs to have this as part of it's if statement instead of exposing a different way to copy prefixes.

Reply:
ok